docs/sorting-algorithms/bubble-sort.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Bubble Sort
+slug: /sorting-algorithms/bubble-sort
+---
+
+## Introduction
+
+Bubble sort is a simple sorting algorithm which compares two adjacent elements and swap if they are not in the correct position.
+
+## Explanation
+
+**AIM -** Repeatedly swap two adjacent elements if they are in the wrong order.
+
+**WORKING -** It works by iterating the array from the first element to the last element, comparing each pair of the elements, and swapping them if needed.
```

Comment:
Make it: comparing each pair of elements, ...

Reply:
okay